packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -633,6 +635,17 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
                 m.isVisible = !selectedMesh || m == selectedMesh;
             });
     }
+
+    protected _unhoverMeshOnTouchUp(pointerId: number, selectedMesh: AbstractMesh) {
+        const engine = this.gizmoLayer.originalScene.getEngine();
+        const isMobileSafari = engine._badOS || (engine._badDesktopOS && document && "ontouchend" in document);
+
+        // force unhover mesh if not a mouse event
+        if (pointerId > 2 || isMobileSafari) {

```

Comment:
Why check for pointer id bigger than 2? WebXR (emulated) pointers start from 100

Reply:
After testing in https://playground.babylonjs.com/#SG9ZZB#29, it looks like checking native event `pointerType` to see if it is `touch` works for non-Safari and Safari mobile while not letting mouse and xr events past.

Reply:
should be resolved as of 16309d8